crates/rpc-proto/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "miden-rpc-proto"
+version = "0.3.0"
+description = "Miden RPC message definitions"
```

Comment:
nit: I would change the description to "Miden node RPC message definitions"

Reply:
Done